README.md

```diff
@@ -1,63 +1,63 @@
-# [Example WASI proposal]
-
-This template can be used to start a new proposal, which can then be proposed in the WASI Subgroup meetings.
-
-The sections below are recommended. However, every proposal is different, and the community can help you flesh out the proposal, so don't block on having something filled in for each one of them.
-
-Thank you to the W3C Privacy CG for the [inspiration](https://github.com/privacycg/template)!
-
-# [Title]
+# WASI Blob Store
 
 A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI) API.
 
 ### Current Phase
 
-[Fill in the current phase, e.g. Phase 1]
+Phase 1
 
 ### Champions
 
-- [Champion 1]
-- [Champion 2]
-- [etc.]
+- Jiaxiao Zhou
+- Kevin Hoffman
```

Comment:
I will be happy to have you as one of the champions of the wasi-blob-store proposal, @autodidaddict !

I will need to ask Lin / Pat about how to add one to the existing champions list.

Reply:
I asked Lin what the process here had been: we don't require any sort of WASI group process to change proposal champions beyond the existing champions agreeing to it. cc: @ricochet